File: board-admin-backend/src/main/java/com/portSrilanka/board_admin_backend/dto/approval/ApprovalRequest.java
package com.portSrilanka.board_admin_backend.dto.approval;

import com.portSrilanka.board_admin_backend.enums.ApprovalStatus;
import lombok.Data;

@Data
public class ApprovalRequest {
    private Long paperId;
    private ApprovalStatus approvalStatus;
    private String approvalComment;
}

File: board-admin-frontend/frontend/lib/features/approvals/model/approval_request.dart
class ApprovalRequest {
  final int paperId;
  final String approvalStatus;
  final String? approvalComment;

  ApprovalRequest({
    required this.paperId,
    required this.approvalStatus,
    this.approvalComment,
  });

  Map<String, dynamic> toJson() {
    return {
      'paperId': paperId,
      'approvalStatus': approvalStatus,
      'approvalComment': approvalComment,
    };
  }
}
